scripts/plugin-godocs.go

```diff
@@ -53,6 +57,31 @@ func docHTML(text string) string {
 	return buf.String()
 }
 
+func removeDuplicates(array []string) []string {
+	keys := make(map[string]struct{})
+	set := []string{}
+	for _, element := range array {
+		if _, value := keys[element]; !value {
+			keys[element] = struct{}{}
+			set = append(set, element)
+		}
+	}
+	return set
+}
+
+func tags(doc string) []string {
+	tagRegexp := regexp.MustCompile(`@tag\s+(\w+)\s*`)
+	submatches := tagRegexp.FindAllStringSubmatch(doc, -1)
+	if (submatches == nil) {
```

Comment:
nit: I think you can eliminate this check, and everything still works. 0/5 on whether the code is simpler as a result.

Reply:
Ok